### ISSUE TITLE
Time Lost Proto Drake

### ISSUE THREAD

**Issue by punims** [dalaranwow/dalaran-wow#5303]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
So as a person who already has TLPD I can't help but notice ever since the spawn for it and varygosa was shut off, nobody else got it... and it has been roughly 2 years since the spawn time was shut off.

**Expected behaviour**: Tell us what should happen instead
Well, it should spawn, share a spawn timer with varygosa and should have a flight path

**Steps to reproduce the problem**:

1. log on
2. hope for tlpd to spawn
3. die waiting.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, Don't think there's much to be added, I think you might have just forgotten to deal with this issue and it's probably a good time to fix it since most people are done farming icc and are now just farming rare mounts and achievements.

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Coltari** [comment]
has the timer been turned off? this makes me sad if so

**Comment by punims** [comment]
Ye it's been off for around 2 years

**Comment by Rushor** [comment]
duplicate of https://github.com/dalaranwow/dalaran-wow/issues/3142
